### ISSUE TITLE
Chrome Notification: Chrome-Audio-Capturer wants to access all your website-data

### ISSUE THREAD

**Issue by timpulver** [arblast/Chrome-Audio-Capturer#2]
When installing the extension, Chrome-Audio-Capturer asks for strange permissions – is this necessary?

**Comment by arblast** [comment]
Yes, the API that allows the extension to access the audio is what's requiring the permission. It only accesses the audio, but there's nothing I can do since that's what Chrome decided to display for that specific API. A lot of popular extensions like Adblock also have this permission, so I don't think it's something you need to worry about.

**Comment by timpulver** [comment]
Thanks for your quick answer!
I do not recall giving another extensions these permissions. Too bad there is no way to request less permissions.
